source/renderer/app/features/menu-updater/MenuUpdater.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { inject } from 'mobx-react';
+import type { StatelessFunctionalComponent } from 'react';
+import type { StoresMap } from '../../stores';
+import { useFeature } from '../../utils/mobx-features/hooks';
+import MenuUpdaterFeature from './feature';
+import type { MenuUpdaterProps } from './types';
+
+const MenuUpdater: StatelessFunctionalComponent<MenuUpdaterProps> = ({
+  app,
+  children,
+  profile,
+  router,
+  staking,
+  uiDialogs,
+}) => {
+  useFeature(
```

Comment:
You can use this pattern to prevent `MenuUpdaterFeature` to be instantiated in every re-render:

```js
  const [menuUpdaterFeature] = useState(() => {
 new MenuUpdaterFeature({ app, profile, router, staking, uiDialogs })
  });
  useFeature(menuUpdaterFeature);
```